physics/smoke_dust/dep_dry_mod_emerson.F90

```diff
@@ -0,0 +1,431 @@
+!>\file dep_dry_mod.F90
+!! This file is for the dry depostion driver.
+!-------------REVISION HISTORY---------------!
+! XX/XX/XXXX : original implementation (Ravan Ahmadov)
+! 08/17/2023 : modified to follow Emerson et al., (2020) (Jordan Schnell)
+! 08/17/2023 : gravitational settling folowing the coarse pm settling driver (Jordan Schnell)
+
+module dep_dry_emerson_mod
+
+  use machine ,        only : kind_phys
+  use dep_data_mod     ! JLS
```

Comment:
It's best practice to pass constants from the host to the scheme via argument list, rather than importing them from module memory from dep_data_mod.F90. Some of these parameters are already defined by the host and could be passed in.